lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/Node/GenNodeConfig.hs

```diff
@@ -193,19 +193,15 @@ genNodeConfig nodeSegment name genesisFiles clusterEra logCfg = do
 
 controlExperimental :: ClusterEra -> ChangeValue
 controlExperimental = \case
-    BabbageHardFork -> setExperimental False
-    ConwayHardFork -> setExperimental True
+    _ -> setExperimental False
 
 setExperimental :: Bool -> ChangeValue
 setExperimental enabled value = value
     & atKey "ExperimentalProtocolsEnabled" ?~ Bool enabled
     & atKey "ExperimentalHardForksEnabled" ?~ Bool enabled
 
 setHardForksForLatestEras :: ClusterEra -> ChangeValue
-setHardForksForLatestEras clusterEra =
-    case clusterEra of
-        BabbageHardFork -> setHardFork (T.pack $ show BabbageHardFork)
-        ConwayHardFork ->
```

Comment:
Will this change not make the integration tests run on Conway regardless of setting?

Reply:
yes, you are totally right . Fixing it ...